resources/lib/functions.py

```diff
@@ -1085,7 +1085,7 @@ def PLAY(params):
     play_info["media_source_id"] = media_source_id
     play_info["use_default"] = use_default
     log.info("Sending embycon_play_action : {0}", play_info)
-    send_event_notification("embycon_play_action", play_info)
```

Comment:
Why this change? Are you trying to be more generic?
But this is the main play event for the addon.
If another addon sends this event EmbyCon will try to play the file.
I dont think you can make this generic and allow other addons to send this as it will break this addon.